### DISCUSSION TITLE
Calculate sunset times for a range of dates.

### DISCUSSION THREAD

**Discussion by hidp123** [cosinekitty/astronomy#235] [Q&A] [answered]
Hi, is it possible to calculate the sunset times for a date range, say, a full year or x years?

https://github.com/cosinekitty/astronomy/blob/master/demo/python/riseset.py

**Comment by hidp123** [comment, marked as answer]
Found it:

https://github.com/cosinekitty/astronomy/tree/master/source/python#SearchRiseSet

**Reply by cosinekitty** [reply]
Yes, that is correct. You can keep calling `SearchRiseSet` to iterate through rise/set events for a given body over any desired span of time.
